deploy.sh

```diff
@@ -163,13 +185,15 @@ then
         --profile ${DEVOPS_PROFILE}
     echo "Waiting for stack to be created ..."
     aws cloudformation wait stack-create-complete --profile ${DEVOPS_PROFILE} --region ${REGION} --stack-name ${STACK_NAME}
+
+    template_protection "mgmt" $STACK_NAME $REGION $DEVOPS_PROFILE
```

Comment:
Any reason why termination protection should still be enabled on this stack when `ENV` is `dev`?

Reply:
@jaidisido fair. I think the reasoning I applied here was that the stack is in the DevOps / Management account so I wanted it protected. I'll change it.